Godeps/Godeps.json

```diff
@@ -1887,6 +1887,11 @@
 			"Rev": "6b7015e65d366bf3f19b2b2a000a831940f0f7e0"
 		},
 		{
+			"ImportPath": "github.com/grpc-ecosystem/go-grpc-prometheus/examples/testproto",
```

Comment:
Why do we need `github.com/grpc-ecosystem/go-grpc-prometheus`? I can't find this being used anywhere.